scripts/pfcstat

```diff
@@ -37,6 +34,12 @@ try:
 except KeyError:
     pass
 
+from utilities_common.netstat import ns_diff, STATUS_NA, format_number_with_comma
+from utilities_common import multi_asic as multi_asic_util
+from utilities_common import constants
+from utilities_common.cli import UserCache
```

Comment:
Check why I moved

Reply:
To run test separatelly from other tests. Looks like when running all tests, previous tests already do the UT monkey patching but running this test individually fails.